package.json

```diff
@@ -294,6 +294,8 @@
     "xpath": "0.0.27"
   },
   "scripts": {
+    "build-backend": "mvn clean install",
```

Comment:
It is used by the command below. 
`npm run backend`
Do you think I should I remove them? They are confusing.

My initial idea was to create a script `npm run dev` that starts both the front-end and the back-end, but maybe this can be part of a future PR, when also the proxy server is aligned to local backend